core/src/main/python/wlsdeploy/tool/discover/discoverer.py

```diff
@@ -727,6 +729,15 @@ def _massage_online_folders(lsc_folders):
     return folder_list
 
 
+def get_interface_name(mbean_interface):
```

Comment:
Why does this method exist in two places?  mbean_utils and discoverer

Reply:
This will take me some time to combine the two together. It is not a straight up conversion. Putting on the back burner for now.